cmd/subcmds/synth.go

```diff
@@ -6,6 +6,8 @@ SPDX-License-Identifier: Apache-2.0
 package subcmds
 
 import (
+	"golang.org/x/exp/maps"
+
```

Comment:
How about upgrading to go 1.23 and getting rid of experimental package?
See [here](https://pkg.go.dev/maps#Keys)